### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by Ridwanullahi-code**
- Implement the design from the sneak peek wireframes.
- Styling is nice to have but focus on making sure everything functions first. (Note: Ignore login logic, you would be using Devise in coming milestone)

- Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).
- Hint: In order to see some posts or comments, you might need to add some data to your database as described [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md#use-models-to-insert-data).

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,12 @@
...
+	<div class="align">
+		<p><%= @user.bio %></p>
+		<% @user.most_recent_posts.each do |post| %>
+		 <%= render 'shared/post', post:post %>
+		<%end %>
+	</div>
+	<div>
+		<button><%= link_to "see all posts", user_posts_path(@user) %></button>
+	</div>
+</section>
```
Comment (Whoistolu) [inline app/views/users/show.html.erb]: - [ ] [REQUIRED] Great job on the User show page so far @Ridwanullahi-code  👍🏽 However, we need to put in more effort to look like the wireframe provided.
### Your Design
Screenshot 2023-04-15 at 19 09 00

### Design Template
Screenshot 2023-04-15 at 19 16 00

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,13 @@
...
+	  <%= render 'shared/post', post: post %>
+		<% post.most_recent_comments.each do | comment | %>
+			<% @user = User.find(comment.author_id) %>
+			<p><%= @user["name"] %>: <span><%= comment.text %></span></p>
+		<% end %>
+	<% end %>
+	<%= content_tag :div, class: "will-paginate-container" do %>
+	<%= will_paginate @posts %>
+	<% end %>	
+</section>
```
Comment (Whoistolu) [inline app/views/posts/index.html.erb]: -  [ ] [REQUIRED] Let's do some finishes also on the user's post index page after clucking the  **See all Post** button. Let's ensure it looks like the template provided.
### Your Design
Screenshot 2023-04-15 at 19 28 37

### Design Template
Screenshot 2023-04-15 at 19 22 31

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @Ridwanullahi-code 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -1 +1,15 @@
...
+	<div class="align">
+		<div class="border">
+			<h2>Bio</h2>
+			<p class="bio"><%= @user.bio %></p>
+		</div>
+		<% @user.most_recent_posts.each do |post| %>
+		 <%= render 'shared/post', post:post %>
+		<%end %>
+	</div>
+	<div class="center">
```
Comment (tufoinnkuo10) [inline app/views/users/show.html.erb:12]: - [optional] Great job.Though everything works well always follow the wireframe design as much as possible. you can centralize the buttons, like the `see all post` and `paganization` so they follow the wireframe. nice work.

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Ridwanullahi-code 

Your project is complete! Great job🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- wireframe structure followed
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**